packages/mysql.rb

```diff
@@ -93,4 +84,10 @@ def self.postremove
         -DWITH_UNIT_TESTS=OFF" # Since we don't currently run the tests, there's no point in building them.
   # 52/136 Test  #52: merge_small_tests ..................................***Failed   62.42 sec
   # run_tests
+  cmake_install_extras do
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/etc/bash.d"
+    File.write "#{CREW_DEST_PREFIX}/etc/bash.d/01-mysql", <<~EOF
+      [ -x #{CREW_PREFIX}/bin/mysql.server ] && #{CREW_PREFIX}/bin/mysql.server start
+    EOF
+  end
```

Comment:
Was this built with the lowest glibc container?

Reply:
It was built on the M90 container, which has glibc 2.27

Reply:
Excellent.